language-design.md

```diff
@@ -105,7 +135,10 @@ you choose to include natural language in the design of a DSL? If so, how might
 you do so? If not, why not?
 
 **Response**
+I do not think they are at odds with eachother. Most people undeniably prefer to use natural language. However, the problem arises when they start to confuse a pseudo-natural language with actual natural language. Computers are not yet advnaced enough to actually interpret natural language so when you try to put natural language into a programming language, you are basically lying to the user. You are saying, I can understand natuaral language when really you cannot. This can cause the user errors. I most likely would not use natural language in the design of a DSL for exactly this reason. -Daniel
 
+I don't think they're necessarily at odds with each other, but really what makes a natural-like programming language easy or difficult depends a lot on how much it looks like natural language. AppleScript, apparently, looks too much like natural language even though it is much less flexible. On the other hand, one of the things I like about Python is that code which would be "abstruse" in other languages (say, "for(int i = 0; i < foo.length; i++)" in Java) is more like natural language
+(e.g. "for f in foo"). But Python is still recognizably not a natural language, and maybe that helps. I guess the short version is that I basically agree with Daniel. -Aaron
 
```

Comment:
I really liked the argument Daniel made. Our group said the two articles were at odds with one another because at it's core it seemed to us as if one article was arguing for natural language in programming languages and the other was arguing the opposite. However, you recognized that the two articles were two distinct ideas: pseudo-natural language in programming languages and natural language in programming languages. In particular, I liked the argument "Computers are not yet advanced enough to actually interpret natural language so when you try to put natural language into a programming language, you are basically lying to the user," because it is so true. Natural language processing is still a huge field of research and to call a language built on "natural language" would indeed cause a lot of errors, as AppleScript showed.